src/stateDirectives.js

```diff
@@ -244,8 +244,15 @@ function $StateRefActiveDirective($state, $stateParams, $interpolate) {
             state: state,
             params: newParams
           });
-          update();
```

Comment:
Hm.

* A single `!` - `if state is not predefined, set lazy to true`
* Triple `!!!` - `if state is not predefined, set lazy to true - and cast to bool`

Double `!!` would reverse the desired effect if the state was present in the initial stack, no?

It's getting late, my mind isn't working quite right - but I'm seeing a single failed spec example with the suggested changes (`it('should match on any child state refs`), none of the newly added specs are failing though. I'll go get some fresh air and come back in a bit.